lang/fr/texts/index-pro.html

```diff
@@ -35,6 +35,14 @@ <h2>Trouvez des opportunités d'amélioration de la qualité des produits</h2>
 	<li>Améliorez significativement le Nutri-Score avec une réduction raisonnable du sucre, du sel ou des acides gras saturés.</li>
 	<li>Détectez les produits avec un moins bon profil nutritionnel que les produits similaires du marché.</li>
 </ul>
+  
+  <h2><b>Nouveau!</b>&nbsp;Complétez les données nécessaires au calcul de l'Éco-score avant sa sortie</h2>
```

Comment:
https://www.w3.org/International/questions/qa-b-and-i-tags

Reply:
that would be <em> ?